### PR TITLE
Added petrockblog's xarcade2jstick

### PR DESCRIPTION
**Pull request by substring**
- corresponding controllers.xarcade.enabled in recalbox.conf
- startup xarcade2jstick in S29recalboxsystem

Couldn't find how to make uinput as a dependancy :/

Needs testing ...

### CODE REVIEW

**Comment by digitalLumberjack** [comment]
uinput is a kernel module, you just have to modprobe uinput :)

File: board/recalbox/fsoverlay/etc/init.d/S29recalboxsystem [hunk 1]
```diff
@@ -114,6 +114,14 @@ rb_hostname() {
     fi
 }
 
+rb_xarcade2jstick() {
+    settings_xarcade="`$systemsetting -command load -key controllers.xarcade.enabled`"
+    if [[ "$settings_xarcade" == "1" ]];then
+        recallog "starting xarcade2jstick"
+        start-stop-daemon --background --start --quiet --exec /usr/bin/xarcade2jstick
```
Comment (digitalLumberjack) [inline board/recalbox/fsoverlay/etc/init.d/S29recalboxsystem:121]: modprobe uniput 
here ?

**Comment by substring** [comment]
no one needed to modprobe uinput.

**Comment by digitalLumberjack** [comment]
As you said "Couldn't find how to make uinput as a dependancy :/" i though it was necessary

**Comment by substring** [comment]
at compilation, that's all

**Comment by digitalLumberjack** [comment]
Ok so it uses kernel headers files ?

**Comment by substring** [comment]
yes sir

``` C
#include <linux/input.h>
#include <linux/uinput.h>
```

**Comment by digitalLumberjack** [comment]
ok so you need to add a dependency to the kernel

**Comment by substring** [comment]
You can merge ;)

**Pull request merged**